src/Parse/Assetic/LessCompiler.php

```diff
@@ -1,13 +1,12 @@
 <?php namespace October\Rain\Parse\Assetic;
 
-use Event;
```

Comment:
@bennothommo @jaxwilko @christianWilling  is there value in moving this filter to the Assetic core? I'm not 100% sure what it does differently from the already included Assetic filters.

Reply:
From the looks of it the October one also add's in file version hashing functionality, could merge it in this filter or add in that functionality as an option for the Assetic project?

Reply:
I haven't spent enough time on this filter or the core Assetic filter to understand the difference between those two options @jaxwilko, are you able to explain it in a bit more detail for me?

Reply:
@JaxWilko ping